techmatters/serverless
# CHI-1657: Disable operating hours checks for non prod builds by default

Note: this was kind of a 'rage PR' that I raised after the staging helpline I was trying to debug refused to let me initiate a webchat because it was 'closed for the day'

So it may not be implemented in the most optimal manner, appreciate your input!

## Description

It is extremely frustrating trying to debug an account specific issue in staging only to be blocked out of chat by an operating hours restriction. Operating hours should not be enforced by default for non production builds, only when you want to test that particular functionality

* Disable operating hours checks for non prod builds by default
* With an override option

### Checklist
- N/A New tests added


### Related Issues
CHI-1657

### Verification steps

* Deploy a staging build, check that the channels can be used out of hours
* Deploy a staging build with the override checked, ensure operating hours are enforced
* Deploy a prod build without the override checked, ensure operating hours are enforced
